### PR TITLE
update changes in 5 files for nc-news-6

### PR DESCRIPTION
**Pull request by AndyBeeee**

### CODE REVIEW

File: __tests__/test.test.js [hunk 1]
```diff
@@ -153,5 +153,57 @@ describe('GET /api/articles', () => {
...
 })    
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('Status code 200 returned', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        })
+
+    test('Request responds with comments in an array that have the correct properties', () => {
```
Comment (Slyose) [inline __tests__/test.test.js:165]: * properties and length, since you're testing for that too!

File: __tests__/test.test.js [hunk 2]
```diff
@@ -153,5 +153,57 @@ describe('GET /api/articles', () => {
...
+    })
+    test('Request responds with a status code 400 when passed an invalid article_id', () => {
+    return request(app)
+        .get('/api/articles/notAnId/comments')
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toEqual('Bad request')
+        })
+    })
+})
```
Comment (Slyose) [inline __tests__/test.test.js:217]: Great work so far, but what happens in the case of an article that DOES exist, but has no comments? (a 200) Test for this case and make sure your server can handle it!

**Review by Slyose — COMMENTED** [review]
Nicely done so far! You've missed one test case though - please implement this and resubmit another PR when done

**Review by mezzDavies — APPROVED** [review]
Looks good Andy 👍 Please merge away!

**Pull request merged**